Doc/library/types.rst

```diff
@@ -362,6 +361,8 @@ Additional Utility Classes and Functions
 
    .. versionadded:: 3.3
 
+   .. versionchanged:: 3.9
+      Attributes are no longer sorted in the repr.
```

Comment:
```suggestion
      Attribute order is no longer preserved in ``__repr__``
```

Reply:
As per the discussion attribute order is preserved in `__repr__` here by removing sorting.

Reply:
Yeah, I think that's right. I think we can rename this as 

``the insertion order of attribute is preserved``